content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -65,11 +65,11 @@ It defines an index.php page which performs some CPU intensive computations:
 First, we will start a deployment running the image and expose it as a service:
 
 ```shell
-kubectl run php-apache --image=k8s.gcr.io/hpa-example --requests=cpu=200m --limits=cpu=500m --expose --port=80  --generator=run-pod/v1
+kubectl apply -f https://k8s.io/examples/application/php-apache.yaml
```

Comment:
Hello @kkanska . Thanks for the updates.
Similar to `hpa` configuration file listed on line 472, you could use a `codenew` shortcode to introduce the configuration file before using the file in the kubectl command. Such as:
```
{{< codenew file="application/php-apache.yaml" >}}
```
The other configuration file is in the `hpa` subdirectory and uses the same name:

```
{{< codenew file="application/hpa/php-apache.yaml" >}}
```